### PR TITLE
Feature/tao 7267 jqueryui removal

### PR DESCRIPTION
**Pull request by krampstudio**
Linked to https://oat-sa.atlassian.net/browse/TAO-7267 

What has changed : 
 - Remove jqueryui in the resizer (not used anymore)
 - Repackage only the Drag&Drop feature of jqueryui into an extension dedicated library. 
(the CVE concerns the dialog part of jqueryui : https://nvd.nist.gov/vuln/detail/CVE-2016-7103 / https://nvd.nist.gov/vuln/detail/CVE-2010-5312)

How to test : 
 - Open the Item Creator and try to drag and drop interactions (compare with develop, it's buggy)

Dependent PR : 
 - [x] https://github.com/oat-sa/tao-core/pull/2076

### CODE REVIEW

File: views/js/qtiCreator/editor/gridEditor/draggable.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ define([
     'lodash',
     'taoQtiItem/qtiCreator/helper/qtiElements',
     'taoQtiItem/qtiCreator/editor/gridEditor/droppable',
-    'jqueryui'
+    'taoQtiItem/lib/jqueryui_dragdrop'
+    //'taoQtiItem/lib/jquery.ui.draggable.min'
```
Comment (ferenckiss89) [inline views/js/qtiCreator/editor/gridEditor/draggable.js]: ?

File: views/js/qtiCreator/editor/gridEditor/draggable.js [hunk 2]
```diff
@@ -3,7 +3,8 @@ define([
     'lodash',
```
Comment (ferenckiss89) [inline views/js/qtiCreator/editor/gridEditor/draggable.js]: this file does not contain a copyright text

File: views/js/qtiCreator/editor/gridEditor/resizable.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ define([
     'taoQtiItem/qtiCreator/editor/gridEditor/config',
```
Comment (ferenckiss89) [inline views/js/qtiCreator/editor/gridEditor/resizable.js]: copyright text is missing

**Comment by krampstudio** [comment]
@ferenckiss89 my bad I should have created the pr in draft mode.

**Pull request merged**